MapboxNavigation/RouteVoiceController.swift

```diff
@@ -182,12 +177,7 @@ open class RouteVoiceController: NSObject, AVSpeechSynthesizerDelegate, AVAudioP
         // Set recentlyAnnouncedRouteStep to the current step
         recentlyAnnouncedRouteStep = routeProgress.currentLegProgress.currentStep
         
-        fallbackText = spokenInstructionFormatter.string(routeProgress: routeProgress, userDistance: userDistance, markUpWithSSML: false)
-        
-        // If the user is merging onto a highway, an announcement to merge is a bit excessive
-        if let upComingStep = routeProgress.currentLegProgress.upComingStep, routeProgress.currentLegProgress.currentStep.maneuverType == .takeOnRamp && upComingStep.maneuverType == .merge && routeProgress.currentLegProgress.alertUserLevel == .high {
-            return false
-        }
+        fallbackText = routeProgress.currentLegProgress.currentStepProgress.currentSpokenInstruction?.text
```

Comment:
FYI, using plain text here will prevent the AVSpeechSynthesizer fallback from honoring pronunciation overrides coming from OSRM. Support for that feature has been awaiting a review in #624. Now we’ll instead have to parse the SSML source code for `<phoneme>` tags using an XML parser, either the SAX-style XMLParser or a third-party library.

Reply:
Considering we're going to soon preload voice instructions, can we punt on this?

Reply:
Yes, hence the FYI. Preloading voice instructions is sort of orthogonal to this discussion, since AVSpeechSynthesizer announces the entire instruction, including freeform parts like the road name. However, we hope to make the fallback occur so infrequently that it’ll be an edge case, and mispronunciation of road names is only a small part of what makes the AVSpeechSynthesizer fallback jarring to the user.